about/contributing_datasets.md

```diff
@@ -0,0 +1,288 @@
+
+This particular example imports the COVID Tracking Project's [historical state dataset](https://covidtracking.com/api/).
+
+The dataset contains various statistics about COVID-19 testing results, including numbers of patients hospitalized, patients in the ICU, patients on ventilators, and patient outcomes.
+
+The end goal of this example is to represent each data point as a `StatVarObservation` like this:
+
+```
+Node: Count_MedicalTest_COVID_19_Pending_geoId/04_2020-03-04
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:Count_MedicalTest_COVID_19_Pending
+observationAbout: dcid:geoId/04
+observationDate: "2020-03-04"
+value: 24
+```
+
+For more information on the terms [`StatisticalVariable`](https://browser.datacommons.org/kg?dcid=StatisticalVariable) and [`StatVarObservation`](https://browser.datacommons.org/kg?dcid=StatVarObservation), see ["Intro to `StatisticalVariable` and `StatVarObservation`"](https://github.com/datacommonsorg/data/tree/master/docs/representing_statistics.md).
+
+### Roadmap
+
+Importing data requires three main steps:
+
+1. Defining new `StatisticalVariable` nodes.
+2. Cleaning the CSV files.
+3. Defining a mapping from CSV to MCF (aka [Template MCF](https://github.com/datacommonsorg/data/tree/master/docs/mcf_format.md#template-mcf)).
+
+This tutorial explains each step in detail and demonstrates where you can submit artifacts for Data Commons review.
+
+## Step 1: Define new `StatisticalVariable` nodes.
+
+**Note:** If the dataset does not introduce new variables, skip this step and refer to existing `StatisticalVariable` identifiers.
+
+### Required triples
+
+At a minimum, each `StatisticalVariable` needs a `populationType` and a curated, human readable identifier. Here's an example:
+
+```
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+```
+
+In this example, "`Count_MedicalTest_COVID_19_Pending`" is the curated identifier. Data Commons recommends making these IDs descriptive of the stat. For example, if your data is specifically for median income among females age 12 and over, you could assign "MedianIncome_Person_Female_12YearsAndOlder".
+Here "`MedicalTest`" is the common type of the group being observed. E.g. "Person" and "Household" if measuring median income of people and households.
+
+### Optional triples
+
+The `statType` and `measured_property` fields can be updated later. If they are not provided, Data Commons will assign `measuredValue` as the `statType` and use the `dcid` as a placeholder `measured_property`. Here is an example that includes these optional triples:
+
+```
+Node: dcid:Count_MedicalTest_COVID_19_Pending
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalTest
+statType: dcs:measuredValue
+measuredProperty: dcs:count
+```
+
+Other common `statType`s include `medianValue`, `meanValue` and `minValue`. Other common `measuredProperty`s include `count`, `incrementalCount`, and `age`.
```

Comment:
Btw, these are all nodes in the graph as well e.g. http://datacommons.org/browser/statType
Do you think it'll be helpful to add a link to them?

Reply:
I definitely would support adding links. Would you like me to link only on the first occurrence on a particular term or for all uses of that term throughout the document?